Changes

```diff
@@ -312,7 +317,6 @@ OCaml 4.10.0
   (Gabriel Scherer, review by Florian Angeletti)
 
 ### Standard library:
-
```

Comment:
Minor nitpick: this line removal shouldn't be in the patch. (But it's fine if we merge with it, I can fix the Changes afterwards.)

Reply:
Fixed. Thanks for noticing.